### DISCUSSION TITLE
Adding a right align feature for Flex and/or Stack components

### DISCUSSION THREAD

**Discussion by Pham-dev** [twilio-labs/paste#2816] [Ideas]
Hi team.  Just looked at the documentation but stack or flex does not have a css attribute where I can right align something
I want to be able to replicate this: "align-items: flex-end".  Right now theres no such css to do so.  I believe this should be pretty standard for design systems but let me know if that exists or if theres a work around for it.

**Comment by SiTaggart** [comment]
Hi @Pham-dev 

Yeah, we were recently looking at Stack specifically with some improvements, which I've picked back up again from this PR. https://github.com/twilio-labs/paste/pull/2728 It should solve the Stack justify problem you describe.

In regard to the Flex component, we need to do a big refactor of that API. It's too opinionated, and too inflexible. If you find yourself reaching for that, I'd suggest just reaching for `Box` with `display="flex"` and just go for it with just flexbox css properties. It will give you the fullest control.
